Tella/Components/Reports/ReportsPageView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ReportsPageView.swift
+//  Tella
+//
+//  Created by Gustavo on 31/07/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ReportsPageView: View {
```

Comment:
sorry @ValbuenaG I didn't check this view carefully if it can be a shared component or not.
I don't think we can use `ReportsPageView`(ConnectionPageView) as a shared component because we have different type of views and parameters, so could you keep the old view in `ReportsView` and just fix the view in `UwaziView`

Reply:
yeah, you are right! I roll back to the previous view. My idea was to reuse the component in both and future connections but each connection has its own view model and specific stuff.